modules/core/lib/Auth/Process/AttributeMap.php

```diff
@@ -89,7 +89,8 @@ private function loadMapFile($fileName)
             }
             $filePath = Module::getModuleDir($m[0]).'/attributemap/'.$m[1].'.php';
         } else {
-            $filePath = $config->getPathValue('attributenamemapdir', 'attributemap/').$fileName.'.php';
+            $attributenamemapdir = $config->getPathValue('attributenamemapdir', 'attributemap/') ?: 'attributemap/';
+            $filePath = $attributenamemapdir.$fileName.'.php';
```

Comment:
`getPathValue()` is broken imho.. It will produce different output for a config-setting set to `null` and a config-setting that is not set at all..

Reply:
Yeah, that's unfortunate. We should probably fix that behaviour, but it's also kind of self-inflicted pain, since the default value can be of any type. In any case, fixing that would be a bit tricky and require more work to make sure nothing breaks, I think...

Reply:
Agreed, so that's why I went with this solution.. Pretty? No! Works? Yes!